floodwatch/src/js/components/Generate.js

```diff
@@ -0,0 +1,73 @@
+import d3 from 'd3';
+// import _ from 'lodash';
+import React, { Component } from 'react';
+import { Row, Col } from 'react-bootstrap';
+import { Chart } from './Chart';
+import {render} from 'react-dom';
+import url from 'url';
+
+import {getVisibilityMap, generateDifferenceSentence, createSentence} from './comparisontools';
+import type {VisibilityMap} from './Compare'
+
+
+export class Generate extends Component {
+blank() {
```

Comment:
Right--so, the compare page feeds in a bunch of mouse functionality, so that mouseovers/outs/etc. all work ([e.g.](https://github.com/O-C-R/floodwatch/blob/203744a3942ca013e6eb4ef589524e07ff2dc60b/floodwatch/src/js/components/Compare.js#L330-L331)). The generate page obvs doesn't need that, but it got cranky if I didn't feed it a function, or tried to give it null. I'm sure there's a better way than a blank function, though.